maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/orchestration/MeansAssessmentOrchestrationServiceTest.java

```diff
@@ -333,4 +343,50 @@ void givenSuccessfulPostAssessmentProcessing_whenCreateIsInvoked_thenRollbackIsN
                 repOrderDTO);
         verify(meansAssessmentService, times(0)).rollback(any());
     }
+
+    @Test
+    void givenARequestWithC3NotEnabled_MattPostProcessIsEnabled_thenPostProcessorShouldCalled() {
+        when(maatCourtDataService.invokeStoredProcedure(any(ApplicationDTO.class), any(UserDTO.class),
+                any(StoredProcedure.class)
+        )).thenReturn(workflowRequest.getApplicationDTO());
+
+        when(featureDecisionService.isC3Enabled(workflowRequest)).thenReturn(false);
+        when(featureDecisionService.isMaatPostAssessmentProcessingEnabled(workflowRequest)).thenReturn(true);
+        when(maatCourtDataApiService.getRepOrderByRepId(anyInt())).thenReturn(repOrderDTO);
+        when(applicationTrackingMapper.buildForAssessmentFlow(any(WorkflowRequest.class), any(RepOrderDTO.class)))
+                .thenReturn(new ApplicationTrackingOutputResult());
+        when(contributionService.calculate(workflowRequest))
+                .thenReturn(applicationDTO);
+
+        orchestrationService.create(workflowRequest);
+
+        verify(contributionService).calculate(workflowRequest);
+        verify(incomeEvidenceService).mangeIncomeEvidence(workflowRequest, repOrderDTO);
+
+        verify(proceedingsService).determineMagsRepDecision(workflowRequest);
+        verify(catDataService, times(0)).handleEformResult(any());
+    }
+
+    @Test
+    void givenARequestWithC3NotEnabled_MattPostProcessIsEnabled_thenHandleEformResultShouldCalled() {
```

Comment:
Should follow the `given_when_then` pattern. There only seems to be two lines difference between the tests. I don't think both will be required given @venkat980 's earlier comments.